WordPress/src/main/res/xml/site_settings.xml

```diff
@@ -290,11 +290,43 @@
             android:title="@string/start_over"
             app:longClickHint="@string/site_settings_start_over_hint" />
 
-        <Preference
-            android:id="@+id/pref_delete_site"
-            android:key="@string/pref_key_site_delete_site"
+        <PreferenceScreen
```

Comment:
I didn't know you could create the screens like this, I am still trying to wrap my head around it, but it looks cool!